system/lib/compiler-rt/ubsan-minimal.symbols

```diff
@@ -0,0 +1,79 @@
+-------- d .L.str.22
+-------- d .L.str.24
+-------- d .L.str.26
+-------- d .L.str.28
+-------- d .L.str.30
+-------- d .L.str.32
+-------- d .L.str.34
+-------- d .L.str.36
+-------- d .L.str.38
+-------- d .L.str.4
+-------- d .L.str.40
+-------- d .L.str.42
+-------- d .L.str.44
+-------- d .L.str.6
+-------- d .L.str.8
+-------- d _ZL10caller_pcs
+-------- d _ZL13caller_pcs_sz
+-------- t _ZL17report_this_errorPv
+-------- t _ZL18abort_with_messagePKc
+-------- t _ZL7messagePKc
+-------- T __ubsan_handle_add_overflow_minimal
+-------- T __ubsan_handle_add_overflow_minimal_abort
+-------- T __ubsan_handle_alignment_assumption_minimal
+-------- T __ubsan_handle_alignment_assumption_minimal_abort
+-------- T __ubsan_handle_builtin_unreachable_minimal
+-------- T __ubsan_handle_cfi_check_fail_minimal
+-------- T __ubsan_handle_cfi_check_fail_minimal_abort
+-------- T __ubsan_handle_divrem_overflow_minimal
+-------- T __ubsan_handle_divrem_overflow_minimal_abort
+-------- T __ubsan_handle_float_cast_overflow_minimal
+-------- T __ubsan_handle_float_cast_overflow_minimal_abort
+-------- T __ubsan_handle_function_type_mismatch_minimal
+-------- T __ubsan_handle_function_type_mismatch_minimal_abort
+-------- T __ubsan_handle_implicit_conversion_minimal
+-------- T __ubsan_handle_implicit_conversion_minimal_abort
+-------- T __ubsan_handle_invalid_builtin_minimal
+-------- T __ubsan_handle_invalid_builtin_minimal_abort
+-------- T __ubsan_handle_load_invalid_value_minimal
+-------- T __ubsan_handle_load_invalid_value_minimal_abort
+-------- T __ubsan_handle_missing_return_minimal
+-------- T __ubsan_handle_mul_overflow_minimal
+-------- T __ubsan_handle_mul_overflow_minimal_abort
+-------- T __ubsan_handle_negate_overflow_minimal
+-------- T __ubsan_handle_negate_overflow_minimal_abort
+-------- T __ubsan_handle_nonnull_arg_minimal
+-------- T __ubsan_handle_nonnull_arg_minimal_abort
+-------- T __ubsan_handle_nonnull_return_minimal
+-------- T __ubsan_handle_nonnull_return_minimal_abort
+-------- T __ubsan_handle_nullability_arg_minimal
+-------- T __ubsan_handle_nullability_arg_minimal_abort
+-------- T __ubsan_handle_nullability_return_minimal
+-------- T __ubsan_handle_nullability_return_minimal_abort
+-------- T __ubsan_handle_out_of_bounds_minimal
+-------- T __ubsan_handle_out_of_bounds_minimal_abort
+-------- T __ubsan_handle_pointer_overflow_minimal
+-------- T __ubsan_handle_pointer_overflow_minimal_abort
+-------- T __ubsan_handle_shift_out_of_bounds_minimal
+-------- T __ubsan_handle_shift_out_of_bounds_minimal_abort
+-------- T __ubsan_handle_sub_overflow_minimal
+-------- T __ubsan_handle_sub_overflow_minimal_abort
+-------- T __ubsan_handle_type_mismatch_minimal
+-------- T __ubsan_handle_type_mismatch_minimal_abort
+-------- T __ubsan_handle_vla_bound_not_positive_minimal
+-------- T __ubsan_handle_vla_bound_not_positive_minimal_abort
+         U abort
+         U emscripten_return_address
+         U strlen
+         U write
```

Comment:
What is this file for?

Reply:
I converted ubsan to be a system library instead of just `libraries_to_link`, and that requires symbol files. I did the same thing for the minimal runtime to be consistent, and this is the result.

Reply:
Again, I think if we revert to the old behavior of linking if and only if the user specific `-fsantize` we can avoid these extra symbols files.   Maybe wait for @kripken to respond because I know he explicitly advised you to go this route.